### PR TITLE
added Matlab directory referenced from software.rst

### PR DESCRIPTION
**Pull request by smoe1**

### CODE REVIEW

File: sphinx/software.rst [hunk 1]
```diff
@@ -34,6 +34,12 @@ also available via their `terminal server
 is available for as little as $49.  This is worth buying if you
 think you will use Matlab regularly.
 
+Some useful Matlab scripts and demos will be available in the `Matlab` subdirectory
+of the :ref:`class_repos`. For demos that use Matlab publiscan use the publish
```
Comment (rjleveque) [inline sphinx/software.rst]: Typo: publiscan

**Review by rjleveque — CHANGES_REQUESTED** [review]
Found a typo.

**Comment by rjleveque** [comment]
Hi Scott,

Did you mean to close this PR?  It hasn't been merged.

**Comment by smoe1** [comment]
Yeah i thought i sent a different pull request. I changed the
software.rates file a bit.

**Pull request closed without merging**